src/neptune/new/exceptions.py

```diff
@@ -313,7 +313,10 @@ def __init__(self, container_id: str, container_type: ContainerType):
         self.container_id = container_id
         self.container_type = container_type
         super().__init__(
-            "{} with ID {} not found. It may have been deleted.".format(container_type.value.capitalize(), container_id)
+            "{} with ID {} not found. It may have been deleted. "
+            "You can use the 'neptune clear' command to delete junk objects from local storage.".format(
+                container_type.value.capitalize(), container_id
+            )
```

Comment:
Not 100% sure about the formatting here. Please alert me (or fix directly) if it's off.